Source/Blazorise.AntDesign/AntDesignStyleProvider.cs

```diff
@@ -51,4 +51,14 @@ public override string ModalBackdropZIndex( int modalOpenIndex )
     #region Layout
 
     #endregion
+
+    #region Offcanvas
+
+    public override string OffcanvasAnimationDuration( int animationDuration )
+        => $"transition-duration: {animationDuration}ms";
+
+    public override string OffcanvasBodyMaxHeight( int maxHeight )
+        => maxHeight > 0 ? $"max-height: {maxHeight}px; overflow-y: auto;" : null;
```

Comment:
@steal4life Can you explain where this feature came from?

Reply:
I saw something similar in a modal, so I added it while I was working on offcanvas, but later I realized I might not need it, so I commented it out... But I think it would be interesting to add, because that way I could allow users to decide on the length of off-screen animation...

Here is code where I saw it first time:

![image](https://github.com/Megabit/Blazorise/assets/79018897/44dc9382-53c1-4c72-88aa-cdbf26411dfd)

Reply:
Ah, I was under the impression David is asking about OffcanvasBodyMaxHeight. My bad.

Regarding the animation duration part, I have fixed that behavior. We can only decide whether to make it with CSS variables instead of inline styles. I think with inline styles we're "more sure" that the animation duration will have higher CSS specificity.

Reply:
definitely, even if maybe in the future we add a parameter that allows the user to specify the length of the animation, I think it would make our offcanvas different from others

Reply:
I was Indeed asking about the max height

Reply:
Now that I think about it, having MaxHeight would mess with the footer. I will need to adjust the styles for it.

Reply:
I have tested it. We will have problems with the footer if OffcanvasBody has `max-height`. Offcanvas is a flex container, and OffcanvasBody needs to fill the entire height for the footer to be stuck at the bottom. 

The alternative is to make the footer `position: absoulute`. But then when I resize the browser, the footer goes over offcanvas if there is not enough space. It looks weird.

Reply:
That settles it. I have removed it from the code.